pkg/api/norman/server/managementstored/setup.go

```diff
@@ -307,6 +304,8 @@ func Clusters(schemas *types.Schemas, managementContext *config.ScaledContext, c
 		clusterValidator.CisBenchmarkVersionLister = managementContext.Management.CisBenchmarkVersions(namespace.GlobalNamespace).Controller().Lister()
 	}
 
+	schema.ActionHandler = handler.ClusterActionHandler
```

Comment:
#### Note for Future
This change is necessary `handler.ClusterActionHandler` is a [struct receiver](https://github.com/rancher/rancher/blob/3edb3679213452fa69956cad7a60db066d5e230f/pkg/api/norman/customization/cluster/actionhandler.go#L39), not a pointer receiver. 

Even though `schema.Validator` is a [pointer receiver](https://github.com/rancher/rancher/blob/3edb3679213452fa69956cad7a60db066d5e230f/pkg/api/norman/customization/cluster/validator.go#L43), it was moved down for consistency.